modules/store/spec/types/feature_creator.spec.ts

```diff
@@ -254,9 +254,7 @@ describe('createFeature()', () => {
         let store!: Store;
         const counterState$ = store.select(selectCounterState);
         const count$ = store.select(selectCount);
-      `).toFail(
-        /Type 'object' is not assignable to type '{ counter: { count: number; }; }'/
-      );
+      `).toFail(/No overload matches this call/);
```

Comment:
The name of this test should be: `it should **not** allow ...` 😅

```suggestion
      `).toFail(/Type 'Record<string, unknown>' is not assignable to type '{ counter: { count: number; }; }'/);
```

Reply:
Thanks, it's been updated.